# 1: single-stick mode
# 2: dual-stick mode
#  for example,
#   3: oota mode
#   4: kobayashi mode
#  are possible
int64 js_mapping_mode

# True: forward(nomal)
# False: reverse
bool direction_flag

# for switching joystick (for future)
string main_controller_name

# 1: normal (nothing special)
# 2: home
# 3: reset
# 4: shutdown
int64 arm_mode

---






